lang/src/org/partiql/lang/eval/physical/operators/DefaultOperatorFactories.kt

```diff
@@ -24,9 +24,11 @@ import org.partiql.lang.planner.transforms.DEFAULT_IMPL_NAME
  * @see [org.partiql.lang.planner.PlannerPipeline.Builder.addRelationalOperatorFactory]
  */
 internal val DEFAULT_RELATIONAL_OPERATOR_FACTORIES = listOf(
+
     AggregateOperatorFactoryDefault,
     SortOperatorFactoryDefault,
     UnpivotOperatorFactoryDefault,
+    SortBasedWindowOperator(DEFAULT_IMPL_NAME),
```

Comment:
Had a conversation with @RCHowell regarding how the factories should be created. We can discuss later -- but also see the 3 above for more preliminary info.

Reply:
I saw the changes, I like the new way. If you agree, I would like to delay the change to the next PR though, trying to reduce the overhead on this one.